src/util.iced

```diff
@@ -11,6 +11,16 @@ exports.json_secure_compare = json_secure_compare = (a,b) ->
 
 ##-----------------------------------------------------------------------
 
+exports.v2_sig_type_from_sig_type = v2_sig_type_from_sig_type = (type) ->
+    # parse out the v2 sig type from a string. i.e. "team.settings -> 46
```

Comment:
spacing is wrong here

Reply:
please set your editor to use only 2 spaces in ICS

Reply:
weird i thought it was set..